myokit/formats/sbml/_importer.py

```diff
@@ -96,53 +102,72 @@ def model(self, path):
         refs = {}
 
         # Handle notes, if given
-        x = dom_child(xmodel, 'notes')
+        x = xmodel.find(self.ns['sbml'] + 'notes')
         if x:
             log.log('Converting <model> notes to ascii')
-            model.meta['desc'] = html2ascii(x.toxml(), width=75)
+            model.meta['desc'] = html2ascii(ET.tostring(x, encoding='unicode'),
```

Comment:
I'm guessing that should be encoding='utf-8' ?

Reply:
I chose unicode because any other encoding returns a bit string which then has to be converted back. But I can change that, no prob.

Reply:
It seems to be failing in Python 2.7

Reply:
utf-8 should be unicode too, so not sure what's going on there